.github/workflows/ci.yml

```diff
@@ -14,15 +14,17 @@ jobs:
     timeout-minutes: 5
     strategy:
       matrix:
-        node-version: [16, 18]
+        node-version: [16, 18, 20]
```

Comment:
Let's separate the ci.yml change in the different PR 🙏

Reply:
Oh sorry, was troubleshooting [over here](https://github.com/thgh/bunchee/actions) didn't think about the actions here.

Will do!